docs/getting-started.md

```diff
@@ -0,0 +1,370 @@
+  ```sh
+  kubectl -n authorino apply -f -<<EOF
+  apiVersion: operator.authorino.kuadrant.io/v1beta1
+  kind: Authorino
+  metadata:
+    name: authorino
+  spec:
+    image: quay.io/3scale/authorino:v0.5.0
+    replicas: 1
+    clusterWide: true
+    listener:
+      tls:
+        certSecretRef:
+          name: authorino-server-cert
+    oidcServer:
+      tls:
+        certSecretRef:
+          name: authorino-oidc-server-cert
+  EOF
+  ```
+</details>
+
+<details>
+  <summary><strong>Cluster-wide (without TLS)</strong></summary>
+
+  ```sh
+  kubectl create namespace authorino
+  kubectl -n authorino apply -f -<<EOF
+  apiVersion: operator.authorino.kuadrant.io/v1beta1
+  kind: Authorino
+  metadata:
+    name: authorino
+  spec:
+    image: quay.io/3scale/authorino:v0.5.0
+    replicas: 1
+    clusterWide: true
+    listener:
+      tls:
+        enabled: false
+    oidcServer:
+      tls:
+        enabled: false
+  EOF
+  ```
+</details>
+
+<details>
+  <summary><strong>Namespaced (with TLS)</strong></summary>
+
+  Create the namespace:
+  ```sh
+  kubectl create namespace myapp
+  ```
+
+  Create the TLS certificates – skip this step if you already have certificates and certificate keys created and stored in Kubernetes `Secret`s in the namespace:
+  ```sh
+  git clone --depth 1 --branch v0.5.0 https://github.com/kuadrant/authorino.git && cd authorino
+  make cert-manager # installs https://github.com/jetstack/cert-manager - skip it if already installed
+  make certs AUTHORINO_NAMESPACE=myapp
+  ```
+
+  Deploy Authorino:
+  ```sh
+  kubectl -n myapp apply -f -<<EOF
+  apiVersion: operator.authorino.kuadrant.io/v1beta1
+  kind: Authorino
+  metadata:
+    name: authorino
+  spec:
+    image: quay.io/3scale/authorino:v0.5.0
+    replicas: 1
+    clusterWide: false
+    listener:
+      tls:
+        certSecretRef:
+          name: authorino-server-cert
+    oidcServer:
+      tls:
+        certSecretRef:
+          name: authorino-oidc-server-cert
+  EOF
+  ```
+</details>
+
+<details>
+  <summary><strong>Namespaced (without TLS)</strong></summary>
+
+  ```sh
+  kubectl create namespace myapp
+  kubectl -n myapp apply -f -<<EOF
+  apiVersion: operator.authorino.kuadrant.io/v1beta1
+  kind: Authorino
+  metadata:
+    name: authorino
+  spec:
+    image: quay.io/3scale/authorino:v0.5.0
+    replicas: 1
+    clusterWide: false
+    listener:
+      tls:
+        enabled: false
+    oidcServer:
+      tls:
+        enabled: false
+  EOF
+  ```
+</details>
+
+## Protect a service
+
+The most common integration to protect resources with Authorino is by putting the _upstream_ service that serves the resources (i.e. the _resource server_ to be protected) behind a reverse-proxy enabled with an authorization filter. To do that, start by having you **upstream service deployed and running** in the Kubernetes server. Then, setup [Envoy](https://www.envoyproxy.io) and create an Authorino `AuthConfig` for your service.
+
+### 1. Setup Envoy
```

Comment:
We are already providing basic info and pointing to the official docs. I think we this should be as far as we should go, otherwise it becomes Envoy docs.

Truth is that it will never be enough, considering how many options one has to setup Envoy. Plus, the more we add here, the more to watch and maintain.